test/distributed/fsdp/test_fsdp_grad_acc.py

```diff
@@ -117,12 +117,6 @@ def _test_grad_acc(
                 point to prefetch the next layer's full parameters during the
                 backward pass, if at all.
         """
-        # Gradient accumulation outside `no_sync()` is not currently compatible
```

Comment:
Previously, we were skipping all CPU offloading tests since every tested config included `use_no_sync == False` 😢 

That is why I thought `use_orig_params=True` worked with `CPUOffload(True)`, but we were actually skipping the test.